htdocs/fourn/facture/card.php

```diff
@@ -3095,12 +3095,12 @@ function setRadioForTypeOfInvoice() {
 			print ' <span class="opacitymediumbycolor paddingleft">'.$langs->transnoentities("ReplaceInvoice", $facreplaced->getNomUrl(1)).'</span>';
 		}
 		if ($object->type == FactureFournisseur::TYPE_CREDIT_NOTE) {
-			$facusing = new FactureFournisseur($db);
 			if ($object->fk_facture_source > 0) {
+				$facusing = new FactureFournisseur($db);
 				$facusing->fetch($object->fk_facture_source);
 				print ' <span class="opacitymediumbycolor paddingleft">'.$langs->transnoentities("CorrectInvoice", $facusing->getNomUrl(1)).'</span>';
 			} else {
-				print ' <span class="opacitymediumbycolor paddingleft">'.$langs->transnoentities("CorrectedInvoiceNotFound").'</span>';
+				print ' <span class="opacitymediumbycolor paddingleft">'.$langs->transnoentities("NoInvoiceToCorrect").'</span>';
```

Comment:
NoInvoiceToCorrect seems better on creation, but on edition (after deletion of source invoice for a special reason), old label seems better.

Reply:
@eldy 
It seems to be better but the translated key "CorrectedInvoiceNotFound" is missing in lang files.
What kind of translated value can be used for "CorrectedInvoiceNotFound" in "en_US" lang file ?